wg-manifests/charter.md

```diff
@@ -1,70 +1,32 @@
-# WG Manifests Charter
+# WG Platform/Manifests Charter
 
-This charter adheres to the conventions, roles and organization management
-outlined in [wg-governance].
+This platform/manifests charter describes the working mode / reality / status quo of the last 5 years as of March 2025.
+It tries to be as lean as possible and balance community and commercial interests.
 
 ## Scope
 
-- Provide a catalog (centralized repository) of Kubeflow application manifests.
-- Provide a catalog of third-party apps for common services.
-
-### In scope
-
-#### Code, Binaries and Services
-
-- Maintain tooling to automate copying manifests from upstream app repos.
-- Maintain a catalog that will allow users to install Kubeflow apps and
-  common services easily on Kubernetes, either on the cloud or on-prem, without
-  depending on external cloud services or closed source solutions. Those
```

Comment:
I like this exclusion, because it ensures that we don't use things that are not generically supported.

Reply:
This is covered by "- The default installation shall not contain deep integration with external cloud services or closed source solutions, instead we aim for Kubernetes-native solutions and light authentication and authorization integration with external IDPs"